CONTRIBUTING.md

```diff
@@ -1,56 +1,47 @@
 # Contributing to Cockroach
 
-### Getting and building
+## Getting and Building
 
 1.  Install the following prerequisites, as necessary:
-  - A C++ compiler that supports C++11. Note that GCC prior to 6.0 doesn't
-    work due to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48891
-  - A Go environment with a recent 64-bit version of the toolchain. Note that
-    the Makefile enforces the specific version required, as it is updated
-    frequently.
+  - A C++ compiler that supports C++11. Note that GCC prior to 6.0 doesn't work due to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48891
```

Comment:
My editor wraps at 80. I inserted actual line breaks now. 

If there's a better width to wrap at, let me know.